src/components/Common/BaseVideoLink.js

```diff
@@ -49,16 +49,16 @@ const Link = styled.a`
 const FlexContent = styled.div`
   display: flex;
 `
-const getColorBasedOnBackground = bg => {
-  switch (bg) {
-    case 'dark':
+const getColorBasedOnBackground = themeVariation => {
+  switch (themeVariation) {
+    case theme.variations.dark:
       return {
         opacity: 0.5,
         useLightIcon: true,
         color: theme.colors.white
       }
-    case 'white':
-    case 'grey':
+    case theme.variations.white:
+    case theme.variations.grey:
```

Comment:
I think it's too early to have that assumption (that grey and light "bg" components behave the same way so we can remove one of those). Currently we have situations where we need the `grey` theme, for example. So that would force us to create a `white`, `grey` and `light` variation, so I don't feel it's the best decision for now.

Or were you suggesting something else?

Reply:
So we chatted about this over a call. Basically we don't know, right now, how many "theme variations" we have/need.

Antonas is documenting these as 3 different things. So the assumption that our code should reflect them as 3 different variations is fair.

Then again, it turns out that the 'grey' and 'white' variations are identical, in terms of the styles of a section's children & literally the only thing (at the moment) that changes, is the background colour.

So the question becomes, should we refer to these as 2 themes - 'grey' & 'white' theme? Or should we refer to this as a single 'light' theme & change the background colour based on something else?

I think there are arguments for both right now actually. But what we really need to do is discuss this more in depth with Antonas, rather than just making a decision amongst devs.

So yep, sure, let's leave it as 3 variations for the moment, as this is following design docs :+1: